bats/tests/profile/invalid-locked-k8s-version.bats

```diff
@@ -0,0 +1,52 @@
+load '../helpers/load'
+
+local_setup() {
+    RD_USE_PROFILE=true
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+@test 'create profile' {
+    PROFILE_TYPE=$PROFILE_LOCKED
```

Comment:
You should move this line to `local_setup()` because you set it in almost every test.